cmake/setup_custom_targets.cmake

```diff
@@ -135,23 +56,19 @@ FILE(WRITE ${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/print_info.cmake
 #                     and reruns the configure and generate phases of CMake
 #    rebuild_cache  - rerun the configure and generate phases of CMake
 #
-#    documentation  - ${_description_string} component 'documentation'
```

Comment:
As already mentioned, we use this in a number of places. One is creating the documentation for a release, but I (and presumably others) also use it if I want to see how the documentation comes out for a particular doc piece I've written. An example is when you work on a tutorial program. It would be useful to keep this.